index.html

```diff
@@ -5171,6 +5219,11 @@ <h2 id="AudioWorklet-Examples">
       this.nextUpdateFrame += this.interval;
       this.postMessage({ volume: this.volume });
     }
+
+    // Retain an active reference if the volume is above a threshold,
+    // so that disconnecting inputs does not immediately cause the meter
+    // to stop computing its smoothed value.
+    return this.volume &gt;= VU_METER_MINIMUM;
```

Comment:
I agree. We might want to keep the VU meter example unchanged, and create a new example that can effectively demonstrate the lifetime issue. A custom delay node sounds good to me as well.